app/models/chore.rb

```diff
@@ -1,7 +1,24 @@
 class Chore < ApplicationRecord
+  enum interval_type: { calendar_days: 0, business_days: 1 }
+
   validates :name, presence: true
   validates :interval_days, numericality: { only_integer: true, greater_than: 0 }
 
+  def reset_cycle_date
+    self.last_done = Date.today
+    self.cycle_reset = Time.now
+    case
```

Comment:
can you split this case statement into it's own function. Also, what's the reasoning for returning false and true?